src/Books.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react';
+import Book from './components/Book';
+import BookInput from './components/BookInput';
+
+export default function Books() {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Kindly remove all disabled linter for professional and efficient code

Please check for other occurrences 👍

Reply:
the setList method is yet to be used but is Unduundubly essential as part of the component hence the comment